solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreakerManager.java

```diff
@@ -49,18 +51,6 @@ public CircuitBreakerManager(final boolean enableCircuitBreakerManager) {
     this.enableCircuitBreakerManager = enableCircuitBreakerManager;
   }
 
-  @Override
```

Comment:
Isn't this a breaking change? If someone implemented their own circuitBreakerManager?

Reply:
I was considering keeping this change only for Solr 9? The main challenge is that it is cumbersome to maintain two different formats of definition within the same (solrconfig.xml) file.

Reply:
Ok, if this targets 9.0, then document it and we should be good. Will you then deprecate CircuitBreakerManager as a plugin in 8.10 and remove it in 9.0 in order to not confuse the two?

Reply:
Yes, I will raise a separate PR for marking deprecation.